src/Error/Middleware/ErrorHandlerMiddleware.php

```diff
@@ -184,10 +191,13 @@ protected function logException($request, $exception)
             return;
         }
 
-        $skipLog = $this->getConfig('skipLog');
-        if ($skipLog) {
-            foreach ((array)$skipLog as $class) {
-                if ($exception instanceof $class) {
+        $unwrapped = $exception instanceof PHP7ErrorException ?
+            $exception->getError() :
+            $exception;
+
+        if ($this->getConfig('skipLog')) {
+            foreach ((array)$this->getConfig('skipLog') as $class) {
+                if ($unwrapped instanceof $class) {
```

Comment:
Do tests need to be updated for logging fixes?

Reply:
This is basically just a copy and paste of the old error handler. But  could sure be added.
Would probably be better to follow @saeideng s advice and re-use more common code instead of duplicating in the long run. But for now out of scope of the bugfix.

Reply:
Actually this change isn't necessary at all. `logException()` never get's `Error` instance wrapped inside `PHP7ErrorException`. The wrapping is only done inside `getRenderer()` when passing the exception instance to `ExceptionRender`'s constructor.

Reply:
Are you 100% sure? I am also referring to any outside or custom handler extending the middleware?
With the above change I just wanted to make double sure nothing goes wrong if it ever would get such an exception passed.

Reply:
The unwrapping is surely not needed with current state of middleware code. If some extending middleware needs it (I don't see why) then it's up to them to update their code as required.

Reply:
I think you are mistaken. The incoming one can very well be PHP7+. The wrapping currently only happens inside getRenderer() and thus does not affect the variable inside handleException().
As such you should revert as that code piece is very well required and should stay both for the class itself as for any extension to function properly.